### PR TITLE
Removing terra-app-delegate and terra-modal-manager packages and site…

### PR DESCRIPTION
**Pull request by tbiethman**
… integrations for terra-framework migration.

### Summary
The terra-app-delegate and terra-modal-manager packages are being migrated from terra-core to terra-framework.

terra-framework PR: https://github.com/cerner/terra-framework/pull/40

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ Components in beta stage may include breaking changes, new features, and bug fix
 |--------------------|---------|--------|
 | [terra-alert](https://github.com/cerner/terra-core/tree/master/packages/terra-alert) | [![NPM version](http://img.shields.io/npm/v/terra-alert.svg)](https://www.npmjs.org/package/terra-alert) | :white_check_mark: |
 | [terra-arrange](https://github.com/cerner/terra-core/tree/master/packages/terra-arrange) | [![NPM version](http://img.shields.io/npm/v/terra-arrange.svg)](https://www.npmjs.org/package/terra-arrange) | :white_check_mark: |
-| [terra-app-delegate](https://github.com/cerner/terra-core/tree/master/packages/terra-app-delegate)  | [![NPM version](http://img.shields.io/npm/v/terra-app-delegate.svg)](https://www.npmjs.org/package/terra-app-delegate) | :white_check_mark: |
-| terra-application  | [![NPM version](http://img.shields.io/npm/v/terra-application.svg)](https://www.npmjs.org/package/terra-application) | :x: |
+| terra-app-delegate | [![NPM version](http://img.shields.io/npm/v/terra-app-delegate.svg)](https://www.npmjs.org/package/terra-app-delegate) | :x: |
```
Comment (mjhenkes) [inline README.md]: I vote we just remove reference from the readme (or comment that the package moved), the package isn't actually deprecated, just moved.

Comment (bjankord) [reply README.md]: Yeah, I think we just remove the references from this README file.

**Comment by tbiethman** [comment]
@bjankord, there is no CHANGELOG in the repo root; any ideas to get around the danger script?

**Comment by bjankord** [comment]
@tbiethman Update the readme for terra-site, I believe that should fix it.

**Comment by tbiethman** [comment]
D'oh that makes sense

**Pull request merged**